pandas/tests/plotting/test_misc.py

```diff
@@ -433,3 +433,98 @@ def test_dictionary_color(self):
         ax = df1.plot(kind="line", color=dic_color)
         colors = [rect.get_color() for rect in ax.get_lines()[0:2]]
         assert all(color == expected[index] for index, color in enumerate(colors))
+
+    @pytest.mark.slow
+    def test_has_externally_shared_axis(self):
```

Comment:
could you also pls add `#GH33819` below this line for easy track in the future?

Reply:
There's only one other occurence in the repo without a space between the '#' and the 'G', all the others have a space, so I've added one here.